arcanist/arcanist/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: 
+
+pkgname=('arcanist')
+pkgdesc="Command line interface for Phabricator"
+arch=('i686' 'x86_64')
+url="https://secure.phabricator.com/book/phabricator/article/arcanist/"
+makedepends=('libcrypt-devel' 'libuuid-devel')
+options=('!libtool')
+license=('APACHE')
+source=(https://github.com/phacility/arcanist/archive/master.zip)
+
+prepare() {
+  cd "${srcdir}/arcanist/arcanist"
+
+  autoreconf -fi
+}
+
+build() {
+  cd "${srcdir}/arcanist/arcanist"
+  ./configure --build=${CHOST} \
+      --prefix=/usr \
+      --libexecdir=/usr/lib/arcanist/arcanist/bin \
+      --datadir=/usr/share/arcanist/arcanist \
+      --enable-layout=Debian \
+      ac_cv_header_windows_h=no \
+      ac_cv_header_winsock2_h=no
+
+  make
+  make DESTDIR="${srcdir}/dest" install
+}
+
+/*
+check() {
+  cd "${srcdir}/apr-${pkgver}"
+  make -j1 check
+}
+/*
+
+package_arcanist_arcanist() {
+  depends=('libcrypt' 'libuuid')
+  groups=('libraries')
+
+  mkdir -p ${pkgdir}/usr/{bin,share}
+  cp -f ${srcdir}/dest/usr/bin/*.dll ${pkgdir}/usr/bin/
+  #cp -rf ${srcdir}/dest/usr/share/doc ${pkgdir}/usr/share/
```

Comment:
Let's not add dead code.

Besides, does `make install DESTDIR=${pkgdir}` not work?